ibis/expr/datatypes.py

```diff
@@ -605,6 +606,140 @@ def _equal_part(self, other, cache=None):
         )
 
 
+# GEO Spatial Data Types
+
+class GeoSpatial(DataType):
+
+    __slots__ = ()
+
+    def __repr__(self):
+        name = self.name.lower()
+        if not self.nullable:
+            return '{}[non-nullable]'.format(name)
+        return name
+
+
+class Point(GeoSpatial):
+    scalar = ir.PointScalar
+    column = ir.PointColumn
+
+    __slots__ = 'x', 'y'
+
+    def __init__(self, x, y, nullable=True):
+        if not isinstance(x, numbers.Real):
+            raise TypeError('Point type x must be a real')
+        if not isinstance(y, numbers.Real):
+            raise TypeError('Point type y must be a real')
+
+        super(Point, self).__init__(nullable=nullable)
+        self.x = x
+        self.y = y
+
+    def __str__(self):
```

Comment:
Is this not sufficiently defined by the `DataType` implementation?

Reply:
I think so .. thanks!